File: benchmark/results/java-invocation-inline/run-004/b-baseline/output/com/metaengine/demo/ordering/services/OrderService.java
package com.metaengine.demo.ordering.services;

import java.util.List;
import com.metaengine.demo.ordering.aggregates.Order;

/** OrderService domain service. */
public class OrderService {
    /** Create an Order from a partial input. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Order by id; returns null if not found. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to limit Orders. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Order by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-monolith-opus-script/run-005/a-mcp/output/src/orders/services/order-service.ts
import { Order } from '../aggregates/order';
import { Id } from '../../shared/value-objects/id';

export class OrderService {

  create(input: Partial<Order>): Order { throw new Error('not implemented'); }

  findById(id: Id): Order | null { throw new Error('not implemented'); }

  list(limit: number): Order[] { throw new Error('not implemented'); }

  delete(id: Id): void { throw new Error('not implemented'); }
}
